app.js

```diff
@@ -1,32 +1,45 @@
-const {getTopics, getEndpoints, getArticlesByID, getArticles, getArticleComments} = require("./controllers/topics.controllers")
-const express = require("express")
-const app = express()
+const {
+  getTopics,
+  getEndpoints,
+  getArticlesByID,
+  getArticles,
+  getArticleComments,
+  postArticleComment,
+} = require("./controllers/topics.controllers");
+const express = require("express");
+const app = express();
+app.use(express.json());
 
-app.get('/api/topics', getTopics)
-app.get('/api', getEndpoints)
-app.get('/api/articles/:article_id', getArticlesByID)
-app.get('/api/articles', getArticles)
-app.get('/api/articles/:article_id/comments', getArticleComments)
+app.get("/api/topics", getTopics);
+app.get("/api", getEndpoints);
+app.get("/api/articles/:article_id", getArticlesByID);
+app.get("/api/articles", getArticles);
+app.get("/api/articles/:article_id/comments", getArticleComments);
+app.post("/api/articles/:article_id/comments", postArticleComment);
 
 app.use((err, request, response, next) => {
-    if (err.code === '22P02') {
-        response.status(400).send({message: "Bad Request"})
-    } else {
-        next(err)
-    }
-})
+  if (err.code === "22P02") {
```

Comment:
Perhaps consider extracting your error handling out at this point to stop your app.js getting too 'messy'